llvm/test/CodeGen/NVPTX/wgmma-sm90a-fence.ll

```diff
@@ -0,0 +1,37 @@
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_90a -mattr=+ptx80 | FileCheck %s
```

Comment:
optional: We mostly generate these checks through utils/update_llc_test_checks.py.
But since these are one-liner checks, I think this is fine.